install/startup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Wait for Kubernetes context to be set and ready"
+
+max_retry_count=5
+kube_system_pod_count=$(kubectl get pod -n kube-system | wc -l)
```

Comment:
this will count the header as well like use `--no-headers` to list only the pods